one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -364,6 +377,13 @@ function sendInvitesToStudents() {
   let successRows = [];
   for (let rowIndex = 0; rowIndex < oneToOneData.length; ++rowIndex) {
     let rowNumber = rowIndex + DATA_BEGIN_ROW;
+    let opsRequestFlag = parseInt(activeSheet.getRange(rowNumber, OPS_REQUEST_INDEX + 1).getValue());
+    if (opsRequestFlag !== 1) {
```

Comment:
Please create 3 new constants at the top and use them.
1. FLAG_IGNORE_ROW
2. FLAG_SEND_INVITE
3. FLAG_CANCEL_INVITE

Reply:
- Added the required constants & used them.